fedgov-cv-ISDcreator-webapp/src/main/webapp/js/files.js

```diff
@@ -48,33 +48,33 @@ function loadFile(map, lanes, vectors, laneMarkers, laneWidths, box, errors, sel
         if (msie10 > 0 || msie11 > 0 || msie12 > 0) {
             $('#open_file_modal').modal('show');
             $('#fileToLoad2').one('change', (event)=>{
-              onChange(event,map, lanes, vectors, laneMarkers, box, laneWidths, selected);
+              onChange(event,map, lanes, vectors, laneMarkers, box, laneWidths, selected, controls);
             });
         }
         else {
             $('#fileToLoad').click();
             $('#fileToLoad').one('change',  (event)=>{
-              onChange(event,map, lanes, vectors, laneMarkers, box, laneWidths, selected);
+              onChange(event,map, lanes, vectors, laneMarkers, box, laneWidths, selected, controls);
             });
         }
    }
 }
 
-function onChange(event,map, lanes, vectors, laneMarkers, box, laneWidths, selected) {
+function onChange(event,map, lanes, vectors, laneMarkers, box, laneWidths, selected, controls) {
```

Comment:
missing description (it can be brief similar to other functions in the file)

Reply:
Addressed :https://github.com/usdot-fhwa-stol/connectedvcs-tools/commit/c9ff7b20a2a576b1be6f30d01513057cbdcd1f43